package.json

```diff
@@ -346,6 +345,7 @@
     "marked": "5.1.1",
     "marked-mangle": "1.1.0",
     "memoize-one": "6.0.0",
+    "micro-memoize": "4.1.2",
```

Comment:
Can we maybe use memoize-one instead? Feels a bit weird that we need 2 memo libraries.